openapi/openapi.yaml

```diff
@@ -1067,8 +1067,12 @@ components:
         id:
           type: string
           description: One or two character code to represent city council districts.
-          pattern: '^([0-9]{1,2})$' 
-          example: 25
+          pattern: '^([0-9]{1,2})$'
+          example: '25'
+        order: 
```

Comment:
`order` should be a parameter in the response for `city-council-districts`, rather than a parameter in the cityCouncilDistrict schema. 

When moving it to the response, we'll also want to add it to the list of required fields, otherwise we'll need to to tedious checks for the field every time we want to use it.

Reply:
*property not parameter